koku/reporting/provider/gcp/models.py

```diff
@@ -22,10 +22,7 @@
     "reporting_gcp_cost_summary_by_region_p",
     "reporting_gcp_cost_summary_by_service_p",
     "reporting_gcp_compute_summary_p",
-    "reporting_gcp_compute_summary_by_project_p",
```

Comment:
Let's move the deletions from `UI_SUMMARY_TABLES` to a separate PR and ship to prod before merging these migrations.